### PR TITLE
Mysql wrapper - support async hooks in mysql requests

### PR DESCRIPTION
**Pull request by haimrait**

### CODE REVIEW

**Review by galbash — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "license": "MIT",
   "scripts": {
     "test": "",
-    "lint:js": "eslint --max-warnings=0 ./src/ ./examples/ ./test/unit_tests ./index.js -f table --ext .js --ext .jsx",
+    "lint:js": "eslint --max-warnings=0 ./src/ ./examples/ ./test/unit_tests ./index.js -f table --ext .js --ext .jsxc",
```
Comment (galbash) [inline package.json:21]: Why?

File: src/wrappers/express.js [hunk 2]
```diff
@@ -24,9 +24,11 @@ const { methods } = require('../consts');
 function expressMiddleware(req, res, next) {
     // Check if endpoint is ignored
     utils.debugLog('Epsagon Express - starting express middleware');
+    const originalAsyncId = asyncHooks.executionAsyncId();
 
     if (shouldIgnore(req.originalUrl, req.headers)) {
         utils.debugLog(`Ignoring request: ${req.originalUrl}`);
+        traceContext.destroyAsync(originalAsyncId, true);
```
Comment (galbash) [inline src/wrappers/express.js:31]: I think you should call set reference and then pass the current async I'd (otherwise some stuff won't be caught)

**Review by galbash — CHANGES_REQUESTED** [review]
Small comments

**Review by galbash — DISMISSED** [review]

**Review by tvaintrob — APPROVED** [review]

**Review by galbash — DISMISSED** [review]

**Review by galbash — APPROVED** [review]

**Review by galbash — APPROVED** [review]

**Review by amitlicht — APPROVED** [review]

**Pull request merged**